lib/uksched/include/uk/thread.h

```diff
@@ -496,14 +678,51 @@ struct uk_thread *uk_thread_create_container2(struct uk_alloc *a,
 					      void *priv,
 					      uk_thread_dtor_t dtor);
 
-/*
- * Helper functions for initializing a thread container and setting it
- * as runnable
+/**
+ * Initializes a thread container as bare and sets it as runnable.
```

Comment:
Not sure if the wording "thread container" is the best option. What does it mean? An uninitialized "uk_thread" structure?